README.md

```diff
@@ -0,0 +1,17 @@
+# TiDB Documentation
```

Comment:
If change this to Documentation, we should also change the following short introduction.